_docs/overview/multi-signature-security.md

```diff
@@ -85,13 +85,10 @@ access funds.
 
 Key distribution also has the following disadvantages:
 
-* **Privacy**: Other signatories will have the ability to see your balance.
-Technical details: Every private key needs to be packaged with the multisig
-redemption script (since losing all redemption scripts is just as bad as losing
-all keys). Redemption scripts, however, allow one to view funds. An alternate
-version of this protocol could be created using a different multisig approach
-besides P2SH transactions, which would eliminate the ability of signatories to
-view balances; see Appendix C for details.
+* **Privacy**: All signatories can see the balance via the multisignature redeem script.
```

Comment:
They can see the balance via the cold storage address, even without the redeem script. And perhaps this is a good time to fix [GlacierScript#25](https://github.com/GlacierProtocol/GlacierProtocol/issues/25).

Reply:
Information added in new commit